src/components/Navigator/QuickNavigationModal.vue

```diff
@@ -29,7 +29,7 @@
         <FilterInput
           v-model="userInput"
           class="quick-navigation__filter"
-          placeholder="filter.search-symbols"
+          :placeholder="$t('filter.search-symbols', { technology })"
```

Comment:
The placeholder content is being passed to `$t`, check the `input` inside the `FilterInput`.

I think we should change it so it just applies the attribute, so you can leave this line like it is here.

Reply:
Good catch! Not sure how I missed this when I made the placeholder change 🤦‍♂️

af9810c488cdc72ec8a843ad162382a365987904